google-cloud-bigquery/lib/google/cloud/bigquery/schema.rb

```diff
@@ -371,7 +371,7 @@ def verify_type type
           type = type.to_s.upcase
           unless Field::TYPES.include? type
             fail ArgumentError,
```

Comment:
Would it be better to simply say this instead?

```ruby
fail ArgumentError, "Type '#{type}' not found"
```

Reply:
Sure. 09e93a4